arangod/Pregel/WorkerConductorMessages.h

```diff
@@ -279,4 +256,78 @@ auto inspect(Inspector& f, CollectPregelResults& x) {
       f.field(Utils::executionNumberKey, x.executionNumber),
       f.field("withId", x.withId).fallback(false));
 }
+
+// ---------------------- modern message ----------------------
+
+using MessagePayload =
+    std::variant<LoadGraph, PrepareGlobalSuperStep, RunGlobalSuperStep>;
+using ResponsePayload =
+    std::variant<ResultT<GraphLoaded>, ResultT<GlobalSuperStepPrepared>,
+                 ResultT<GlobalSuperStepFinished>>;
```

Comment:
I know I am being annoying, but could this be a `ResultT<std::variant<GraphLoaded, GlobalSuperStepPrepared, GlobalSuperstepFinished>`?

Reply:
Do also want this if we now smash together both Payloads?
Adding them together we would have 
```
using MessagePayload = std::variant<LoadGraph, PrepareGlobalSuperStep, RunGlobalSuperStep,  
                ResultT<GraphLoaded>, ResultT<GlobalSuperStepPrepared>, ResultT<GlobalSuperStepFinished>>;
```
With your suggestion we would e.g. wrap LoadGraph with an unneccessary ResultT. I don't see a case where it is necessary to wrap LoadGraph in an ResultT.
But perhaps you just want to make it the same for all messages, then we just do this unnecessary wrap, just wanted to mention it.